app-engine/java/ultimate/src/com/google/cloud/tools/intellij/appengine/java/ultimate/impl/AppEngineStandardUltimateWebIntegration.java

```diff
@@ -220,6 +228,34 @@ public void setupDevServer() {
     getOrCreateAppServer();
   }
 
+  /**
+   * Given a non-empty list of {@link WebRoot} tries to locate a "suitable" one and returns its
+   * path.
+   *
+   * <p>First attempts to find a web root with a root relative path to the deployment root, i.e.
+   * "/".
+   *
+   * <p>If none exist with a root relative path, simply grab the first web root and return its path.
+   */
+  @Nullable
+  private String findWebRootPath(List<WebRoot> webRoots) {
+    if (webRoots.isEmpty()) {
+      return null;
+    }
+
+    // TODO test that "/" check works in Windows
```

Comment:
good question :)

Reply:
yep. I'll make sure to do that before merging :)

Reply:
confirmed working in Windows. (The logic is working off of VirtualFiles so the paths are the same).